lib/Models/TerriaJsonCatalogFunction.js

```diff
@@ -443,4 +491,102 @@ TerriaJsonCatalogFunction.prototype.invoke = function() {
   return promise;
 };
 
+// private method split into function to make it recursive
+TerriaJsonCatalogFunction.prototype._handleHttp202 = function(all) {
```

Comment:
This is purely a matter of opinion, but I'm not sure what `all` as a variable name really means here? `response` might be a better choice, maybe.

Reply:
started out with response = {Status:202,Headers:{Object ...},Body:""}, but then switched to mimicking the XMLHTTPResponse object, which then had response.response, so i switched to all. 
all as a variable name was the entire xhr object, as opposed to the response part of the xhr call from Resource options method=GET, or the status and headers from Resource options method=OPTIONS or method=HEAD. 
might make more sense to rename all to xhr?